docker/elasticsearch/Dockerfile

```diff
@@ -1,12 +1,6 @@
-FROM docker.elastic.co/elasticsearch/elasticsearch:5.6.9
+FROM docker.elastic.co/elasticsearch/elasticsearch:5.6.10
 
 RUN bin/elasticsearch-plugin remove x-pack --purge
 
 # Add your elasticsearch plugins setup here
 # Example: RUN elasticsearch-plugin install analysis-icu
-
-USER root
```

Comment:
Historically, we used an unofficial ES image in version 5.5 from hub.docker.com. It uses a different approach to the rights (UID / GID) than in the current, official ES 5.6 and higher images.

Removing this wrapper will result in the loss of backward compatibility when someone tries to jump between our old docker-compose.yml (stack from ES 5.5) and the proposed version.

At best, ES will not start. At the worst, it cleans / loses index data.

Reply:
Since we are not using the same image, I suppose that Docker will create a new container. There is no data loss when the container is deleted because it is on a volume.

Reply:
It will create, you are right. But the volume is still persistent. And both ES images (unofficial 5.5 and official 5.6) use different permissions on it (UID / GID)

Did you test the migration path between older stack version (for example https://github.com/DivanteLtd/vue-storefront-api/blob/5e71f415ccada528d77c5f6b874720f01bb3c0a1/docker-compose.yml) and the proposed one, with index imported (`yarn restore`)?

I'm quite sure that ES won't start in this case (due to different data files permissions).

Reply:
If that's the case, we aren't fixing anything be screwing up the file permissions of all the people who are jumping on the train. BTW, this compose file should only be used for development purposes.

Reply:
We have discussed this with @pkarw for a moment.

We can abandon backward compatibility with "unofficial 5.5" if the compatibility with other official ES images is maintained. So, it'll be not a case anymore. File `wrapper.sh` can be removed.